modules/transformations/redshift/doc/ST_CENTERMEAN.md

```diff
@@ -0,0 +1,23 @@
+### ST_CENTERMEAN
+
+{{% bannerNote type="code" %}}
+transformations.ST_CENTERMEAN(geog)
+{{%/ bannerNote %}}
+
+**Description**
+
+Takes a Feature or FeatureCollection and returns the mean center.
+
+* `geog`: `GEOMETRY` feature to be centered.
+
+
+**Return type**
+
+`GEOMETRY`
+
+**Example**
+
+```sql
+SELECT transformations.ST_CENTERMEAN(ST_GEOMFROMTEXT('POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))'))
+-- POINT (26 24)
```

Comment:
I'm looking to the results of BQ and SF and this is equal to SF but not to BQ. We have to explore what is going on.

Reply:
Here it looks like you are adding the first coordinate twice (it's the same that the last one). That it's what the problem looks to me, but the result still doesn't match other warehouses. I'm checking.